arctic/store/_pandas_ndarray_store.py

```diff
@@ -197,7 +198,7 @@ def append(self, arctic_lib, version, symbol, item, previous_version, **kwargs):
 
     def read(self, arctic_lib, version, symbol, **kwargs):
         item = super(PandasDataFrameStore, self).read(arctic_lib, version, symbol, **kwargs)
-        return self.SERIALIZER.deserialize(item)
+        return self.SERIALIZER.deserialize(item, force_bytes_to_unicode=FORCE_BYTES_TO_UNICODE)
```

Comment:
is it worth allowing this to be specified/overidden as part of the kwargs?

Reply:
I was discussing this with @yschimke whether we should do kwargs args in read vs global, but I assumed the use case was so common that people will just set it as a global once and use it like that. What do you prefer?